src/main/java/chess/domain/board/ChessBoard.java

```diff
@@ -0,0 +1,88 @@
+package chess.domain.board;
+
+import chess.domain.piece.Bishop;
+import chess.domain.piece.Color;
+import chess.domain.piece.King;
+import chess.domain.piece.Knight;
+import chess.domain.piece.Pawn;
+import chess.domain.piece.Piece;
+import chess.domain.piece.Queen;
+import chess.domain.piece.Rook;
+import chess.domain.position.File;
+import chess.domain.position.Position;
+import chess.domain.position.Positions;
+import chess.domain.position.Rank;
+import chess.dto.ChessBoardDto;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class ChessBoard {
+    private final Map<Position, Piece> chessBoard = new LinkedHashMap<>();
+
+    public ChessBoard() {
+    }
+
+    public void initialBoard() {
+        initializeBlackPieces();
+        initializeWhitePieces();
+    }
+
+    public ChessBoardDto convertToDto() {
+        return new ChessBoardDto(chessBoard);
+    }
+
+    public void move(Position source, Position target) {
+        if (!canMove(source, target)) {
+            throw new IllegalArgumentException("올바르지 않은 이동입니다.");
+        }
+        Piece sourcePiece = chessBoard.get(source);
+        chessBoard.put(target, sourcePiece);
+        chessBoard.remove(source);
+    }
+
+    private boolean canMove(Position source, Position target) {
+        if (!chessBoard.containsKey(source)) {
+            throw new IllegalArgumentException("이동할 수 있는 말이 없습니다.");
+        }
+
+        Piece sourcePiece = chessBoard.get(source);
+        Color targetPieceColor = Color.NONE;
```

Comment:
None은 여기서만 쓰이고, 또 NONE 상테는 불가능하지만 문법상 할당이 안 되어있으면 안되니 존재하는 것 같네요.
sourcePiece에서 getColor를 할 수 있는 상황이니, 굳이 None이라는 개념이 없어도 된다고 생각합니다.
```suggestion
        Color targetPieceColor = sourcePiece.getColot();
```

Reply:
각 Piece의 `canMove` 로직을 변경하면서 getColor와 NONE을 삭제한 대신
Piece에 EMPTY 항목을 추가해보았습니다.